tests/keras_autolog/test_keras_autolog.py

```diff
@@ -261,15 +272,24 @@ def test_keras_autolog_log_models_configuration(
     assert ("model" in artifacts) == log_models
 
 
+# In keras 2.6.0, early stopping doesn't work correctly when `patience` is set to 0:
+# https://github.com/keras-team/keras/pull/14750
+patience_values = [1, 5] if keras_version == "2.6.0" else [0, 1, 5]
```

Comment:
Workaround for https://github.com/mlflow/mlflow/pull/4679#issuecomment-899961968